lms/djangoapps/course_api/blocks/transformers/__init__.py

```diff
@@ -50,5 +51,13 @@ def __init__(
         BlockNavigationTransformer,
         requested_field_name='nav_depth',
         serializer_field_name='descendants',
+    ),
+
+    # Provide the staff visibility info stored when VisibilityTransformer ran previously
+    SupportedFieldType(
+        'merged_visible_to_staff_only',
+        VisibilityTransformer,
+        requested_field_name='visible_to_staff_only',
+        serializer_field_name='visible_to_staff_only',
```

Comment:
Nit: Note the `serializer_field_name` parameter is not needed since it defaults to `requested_field_name`.  However, it's fine to keep it in here if you want to be explicit.